### PR TITLE
fix: Dutch adds

### PR DESCRIPTION
**Pull request by aleene**
### Description
More dutch updates

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -47954,6 +47983,13 @@ en:Salted cashews, Salted cashew nuts
 de:Gesalzene Cashewnüsse, Gesalzene Cashewkerne
 fr:Noix de cajou salées
 
+<en:Cashew nuts
+<fr:Fruits à coque salés
+en:Unalted cashews, Unsalted cashew nuts
```
Comment (teolemon) [inline taxonomies/categories.txt:47988]:
```suggestion
en:Unsalted cashews, Unsalted cashew nuts
```

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -86244,7 +86300,7 @@ en:Powdered sugars, Confectioner's sugars, Icing sugars
 de:Puderzucker, Staubzucker, Dekorierzucker
 es:Azúcar glas, Azúcar en polvo, Azúcar glacé
 fi:tomusokerit
-fr:Sucres glaces, Sucres impalpables, Sucres à glacer, Sucre glace
+fr:Sucres glaces, Sucres impalpables, Sucres à glacer, Sucre glace, Sucres en poudre, Sucre en poudre
```
Comment (teolemon) [inline taxonomies/categories.txt:86303]: not quite the same thing

Comment (teolemon) [reply taxonomies/categories.txt:86303]: https://en.wikipedia.org/wiki/Powdered_sugar

Comment (aleene) [reply taxonomies/categories.txt:86303]: From the wikipedia: "Powdered Sugar, also called confectioners' sugar, 10X sugar or icing sugar".

Or are these different in french?

Comment (stephanegigandet) [reply taxonomies/categories.txt:86303]: @aleene It is different in French, "sucre en poudre" is "caster sugar" : a bit finer than "granulated sugar", but much much coarser than icing sugar ("sucre glace").

See http://www.cultures-sucre.com/autour-du-sucre/a-chaque-sucre-son-usage/

(in Canada, "powdered sugar" is translated litterally as "sucre en poudre").

Comment (aleene) [reply taxonomies/categories.txt]: OK reverted

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**